programs/mango-v4/src/instructions/group_edit.rs

```diff
@@ -7,6 +7,7 @@ pub struct GroupEdit<'info> {
     #[account(
         mut,
         has_one = admin,
+        constraint = group.load()?.is_operational()
```

Comment:
I'd not do the check on the _edit instructions. Maybe the group should be halted and can be restarted once a bunch of edits have been made.

Reply:
makes sense, I have removed the constraint in 

group_edit.rs
perp_edit_market.rs
serum3_edit_market.rs
token_edit.rs